nimble/controller/src/ble_ll_conn.c

```diff
@@ -1232,8 +1228,8 @@ ble_ll_conn_event_start_cb(struct ble_ll_sched_item *sch)
 #endif
 
 #if (BLE_LL_BT5_PHY_SUPPORTED == 1)
-        ble_phy_mode_set(connsm->phy_data.rx_phy_mode,
-                             connsm->phy_data.rx_phy_mode);
+        ble_phy_mode_set(connsm->phy_data.tx_phy_mode,
```

Comment:
if I understand it correctly, this call should be done somewhere around line 1191 otherwise phy is not set before TX as a master because this `ble_phy_mode_set()` has been removed from `ble_ll_conn_tx_data_pdu()`
But maybe I miss something - did you test it?